notebooks/3_subset_data.ipynb

```diff
@@ -0,0 +1,1358 @@
+{
```

Comment:
<p>I recommend that you either choose a different number (perhaps Age &gt; 35) or flip the sign (Age &lt; 18). As is, the filter Age &gt; 18 doesn't change the head of the DataFrame, and thus the reader can't see what has changed.</p><p> </p><p>Also, if you introduce the shape attribute (above), you can also show that the shape has changed after the filter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>